go.sum

```diff
@@ -5,6 +5,10 @@ github.com/algorand/avm-abi v0.1.0 h1:znZFQXpSUVYz37vXbaH5OZG2VK4snTyXwnc/tV9CVr
 github.com/algorand/avm-abi v0.1.0/go.mod h1:+CgwM46dithy850bpTeHh9MC99zpn2Snirb3QTl2O/g=
```

Comment:
I'm no `go.sum` expert, but I'd expect references to prior versions of the avm-abi package (0.1.0, 0.1.1, and 0.1.2) to be removed in favor of the latest version

Reply:
Done

Reply:
I think d056257 removed too much, the `github.com/algorand/avm-abi v0.2.0 h1:bkjsG+BOEcxUcnGSALLosmltE0JZdg+ZisXKx0UDX2k=` line should have stayed